libs/sdk-ui-filters/src/AttributeFilter/AttributeFilter.tsx

```diff
@@ -64,8 +69,19 @@ export interface IAttributeFilterProps {
 
     /**
      * Specify a parent attribute filter that will be used to reduce options for available components options.
+     *
+     * Parent filters elements must contain their URIs due to current backend limitations.
+     */
+    parentFilters?: ValuesOrPlaceholders<IAttributeFilter>;
+
+    /**
+     * Specify {@link @gooddata/sdk-ui#IPlaceholder} to use to get and set the value of the attribute filter.
+     *
+     * Note: It's not possible to combine this property with "filter" property. Either - provide a value, or a placeholder.
+     * There is no need to specify 'onApply' callback if 'connectToPlaceholder' property is used as the value of the filter
+     * is set via this placeholder.
      */
-    parentFilters?: IAttributeFilter[];
+    connectToPlaceholder?: IPlaceholder<IAttributeFilter>;
```

Comment:
Why was this not just added to the `filter` prop? So that it would either take a filter or placeholder of it? Is it because of the onApply?

Reply:
I consulted this and we decided to leave this "as is" and to throw an error if used incorrectly.